sdk/openai/Azure.AI.OpenAI/README.md

```diff
@@ -94,53 +137,56 @@ List<string> examplePrompts = new(){
     "Describe in single words only the good things that come into your mind about your mother.",
 };
 
-foreach (var prompt in examplePrompts)
+foreach (string prompt in examplePrompts)
 {
     Console.Write($"Input: {prompt}");
-    var request = new CompletionsOptions();
+    CompletionsOptions request = new CompletionsOptions();
```

Comment:
avoid request / response variable names

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/33789

Reply:
Bad naming.